src/modules/Y2ModuleConfig.rb

```diff
@@ -426,8 +458,29 @@ def resource_aliases_map
     publish :function => :getResourceData, :type => "any (map, string)"
     publish :function => :Deps, :type => "list <map> ()"
     publish :function => :SetDesktopIcon, :type => "boolean (string)"
+    publish :function => :required_packages, :type => "map <string, list> (list <string>)"
     publish :function => :unhandled_profile_sections, :type => "list <string> ()"
     publish :function => :unsupported_profile_sections, :type => "list <string> ()"
+
+  private
+
+    # Returns package name of a given schema.
+    # This information is stored in /usr/share/YaST2/schema/autoyast/rnc/includes.rnc
+    # which will be provided by the yast2-schema package.
+    #
+    # @param schema <String> schema name like firewall, firstboot, ...
+    # @return <String> package name or nil
+    def package_name_of_schema(schema)
+      unless @schema_package
```

Comment:
It might be a matter of personal taste, but I think we also have this in our Rubocop style: use _unless_ in Ruby only for one-liners, otherwise use _if !_ to make it better readable.

Reply:
Also, maybe do a premature _return_ (with the appropriate return value) if the condition is not fulfilled. That saves one level of indentation and complexity.

Again, this may be a matter of personal taste.